reads_filtration/module_wraper.py

```diff
@@ -57,7 +83,7 @@ def run_first_phase(fastq_path, first_phase_output_path, logs_dir, barcode2sampl
                 continue
             for file in os.listdir(dir_path):
                 # look for faa files to collapse
-                if not file.startswith(f'{dir_name}.faa'):  # maybe there's a .gz afterwards
+                if not file.startswith(f'{dir_name}.faa') and not file.startswith(f'{dir_name}_mapitope.faa'):  # maybe there's a .gz afterwards
```

Comment:
We also want unique rpm for mapitope

Reply:
this will create unique rpms for mapitop files.
or do you mean rpm_factors.txt?